assets/src/edit-story/components/canvas/multiSelectionMoveable/index.js

```diff
@@ -191,7 +191,7 @@ function MultiSelectionMoveable({ selectedElements }) {
       ...selectedElements.map((element) => element.id),
       backgroundElement.id,
     ])
-  ).filter(({ isBackground }) => !isBackground);
+  );
```

Comment:
Removing this after discussion with @miina 

These are HTML elements/nodes not "our" elements, so they don't have `isBackground` properties.